components/common/constants.ts

```diff
@@ -0,0 +1,15 @@
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
+
+export const DAI = toAddress('0xdA816459F1AB5631232FE5e97a05BBBb94970c95');
+export const USDC = toAddress('0xa354F35829Ae975e850e23e9615b11Da1B3dC4DE');
+export const USDT = toAddress('0x7Da96a3891Add058AdA2E826306D812C638D87a7');
+export const YFI = toAddress('0xdb25cA703181E7484a155DD612b06f57E12Be5F0');
+export const ETH = toAddress('0xa258C4606Ca8206D8aA700cE2143D7db854D168c');
+export const BTC = toAddress('0xA696a63cc78DfFa1a63E9E50587C197387FF6C7E');
+export const yvWFTM = toAddress('0x0DEC85e74A92c52b7F708c4B10207D9560CEFaf0');
+export const yvUSDC = toAddress('0xEF0210eB96c7EB36AF8ed1c20306462764935607');
+export const yvDAI = toAddress('0x637eC617c86D24E421328e6CAEa1d92114892439');
+export const yvUSDT = toAddress('0x148c05caf1Bb09B5670f00D511718f733C54bC4c');
+export const yvWETH = toAddress('0xCe2Fc0bDc18BD6a4d9A725791A3DEe33F3a23BB7');
+export const yvWBTC = toAddress('0xd817A100AB8A29fE3DBd925c2EB489D67F758DA9');
+export const yvYFI = toAddress('0x2C850cceD00ce2b14AA9D658b7Cad5dF659493Db');
```

Comment:
A bit confusing as they are all vaults. Maybe something like this?

```suggestion
export const ETH_YVDAI = toAddress('0xdA816459F1AB5631232FE5e97a05BBBb94970c95');
export const ETH_YVUSDC = toAddress('0xa354F35829Ae975e850e23e9615b11Da1B3dC4DE');
export const ETH_YVUSDT = toAddress('0x7Da96a3891Add058AdA2E826306D812C638D87a7');
export const ETH_YVYFI = toAddress('0xdb25cA703181E7484a155DD612b06f57E12Be5F0');
export const ETH_YVETH = toAddress('0xa258C4606Ca8206D8aA700cE2143D7db854D168c');
export const ETH_YVBTC = toAddress('0xA696a63cc78DfFa1a63E9E50587C197387FF6C7E');
export const FTM_YVWFTM = toAddress('0x0DEC85e74A92c52b7F708c4B10207D9560CEFaf0');
export const FTM_YVUSDC = toAddress('0xEF0210eB96c7EB36AF8ed1c20306462764935607');
export const FTM_YVDAI = toAddress('0x637eC617c86D24E421328e6CAEa1d92114892439');
export const FTM_YVUSDT = toAddress('0x148c05caf1Bb09B5670f00D511718f733C54bC4c');
export const FTM_YVWETH = toAddress('0xCe2Fc0bDc18BD6a4d9A725791A3DEe33F3a23BB7');
export const FTM_YVWBTC = toAddress('0xd817A100AB8A29fE3DBd925c2EB489D67F758DA9');
export const FTM_YVYFI = toAddress('0x2C850cceD00ce2b14AA9D658b7Cad5dF659493Db');
```

Reply:
Lazy me, refactored with https://github.com/yearn/macarena-finance/pull/38/commits/0890bff873840b2218f5ecd24737c04487614e95